app/javascript/src/components/Dashboard/Contacts/Create.jsx

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+
+import { Pane, Typography } from "neetoui";
+
+import ContactForm from "./Pane/Form";
+
+const ContactPane = ({ showPane, onClose }) => (
```

Comment:
This should be named `Create` itself and import it using custom alias wherever required. The filename and component name should match. Also prefixing with the `Contact` keyword is unnecessary over here. Refer: https://academy.bigbinary.com/learn-react/pages/structuring-code#:~:text=Avoid%20Unnecessary%20Namespacing